arch/arm/boot/dts/overlays/revpi-connect-se-overlay.dts

```diff
@@ -7,13 +7,14 @@
 #include "revpi-connect-overlay.dts"
 
 /{
-	compatible = "brcm,bcm2837";
```

Comment:
I always wondered why we have this compatible here. Have you checked that removing this has no side effects?

Reply:
I tested without it in connect-overlay.dts, it has no diff in dmesg and funtions of piTest.
However, delete it here, it is still contained in connect-se.dtbo, as it is still existed in connect-overlay.dts.

BTW, I checked the code: drivers/of/overlay.c, besides target-path and __overlay__, nothing else has been taken cared.

Reply:
But every overlay I looked at has it. So I guess every overlay just copied it or it has a purpose! So please leave it as is.